src/profile/profile.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/mongoose';
+import { ReturnModelType } from '@typegoose/typegoose';
+import * as cloudinary from 'cloudinary';
+import DataUri = require('datauri/parser');
+import configuration from '~shared/config/configuration';
+
+import { User } from '../users/models/user.entity';
+
+@Injectable()
+export class ProfileService {
+    uploader: typeof cloudinary.v2.uploader;
+
+
+    constructor(@InjectModel(User.modelName)
+    protected readonly userEntity: ReturnModelType<typeof User>) {
+        const { cloudinary: config } = configuration();
+        cloudinary.v2.config({
+            cloud_name: config.name,
+            api_key: config.key,
+            api_secret: config.secret
```

Comment:
Codacy found an issue: [Identifier 'api_secret' is not in camel case.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-backend/pullRequest?prid=5802778)